lib/charms/opensearch/v0/opensearch_peer_clusters.py

```diff
@@ -525,10 +529,10 @@ def rel_data(self) -> Optional[PeerClusterRelData]:
         rel = self._charm.model.get_relation(
             PeerClusterOrchestratorRelationName, orchestrators.main_rel_id
         )
-        if not (data := rel.data[rel.app].get("data")):
+        if not (redacted_dict_str := rel.data[rel.app].get("data")):
```

Comment:
let's keep it simple, and just name it `data` - given that the new data format is with "redacted" secrets